src/app/services/map.service.ts

```diff
@@ -687,7 +701,7 @@ export class MapService {
     this.map.addLayer({
       'id': 'bboxLayer', 'type': 'line', 'source': 'bbox',
       'paint': { 'line-color': '#ea1212', 'line-width': 5, 
-      'line-dasharray': {   "stops": [
+      'line-dasharray': { type: "exponential", "stops": [

```

Comment:
@Binnette According to the type system this needs to be added. Do you spot an issue with this change?

Reply:
https://maplibre.org/maplibre-gl-js-docs/style-spec/layers/#paint-line-line-dasharray

I think this is from a very old version of mapbox gl. Now it's just a simple array [number of full pixels, number of empty pixels]. ex : [ 3, 3 ]

Reply:
Thanks for the review. Again the types of the lib come short in this case. I've reverted my changes and fixed the problem by adding an exemption.